overlays/46-audio-receiver/home/neon/.cache/pip/wheels/33/99/2c/061d0c6934509a9cc08470980d68fbfc972bd22a45c3f306d6/origin.json

```diff
@@ -0,0 +1 @@
+{"archive_info": {"hash": "sha256=665fbe980c91e8b31ad78ed3f66879946948200864002d193f67eccc1d7d5d83", "hashes": {"sha256": "665fbe980c91e8b31ad78ed3f66879946948200864002d193f67eccc1d7d5d83"}}, "url": "https://files.pythonhosted.org/packages/44/80/96d6317a15a13a4f80ffa61118dc144a70756135fbc6ace30f9b033f51f7/PyGObject-3.44.1.tar.gz"}
```

Comment:
Is this cache included intentionally or could these dependencies be downloaded at build time?

Reply:
This is intentional - building these libraries was a hassle and there were no wheels available. Eventually those should go into the binary shop and be built regularly, but for now, this works. You just moved to Python 3.10 recently and the architecture isn't changing, so the thing that would push us to a different version is an update to the library itself.